source/WAVChunk.cpp

```diff
@@ -177,4 +177,28 @@ bool WAVChunk::IsEqual(WAVChunk &wavChunk)
 	bIsEqual = (bBaseEqual && bIsEqual);
 
 	return bIsEqual;
+}
+
+std::shared_ptr<nlohmann::json> WAVChunk::ToJSON()
+{
+	auto JSONDocument = nlohmann::json();
+	auto strChunkName = ChunkTypesNamingUtility::toString(GetChunkType());
+
+	// Adding in Basechunk fields
+	JSONDocument[strChunkName]["SourceIndentifierSize"] = std::to_string(m_u16SourceIndentifierSize);
+	JSONDocument[strChunkName]["SourceIndentifier"] = m_vu8SourceIdentifier;
+
+	// Adding in WAVChunk fields
+	JSONDocument[strChunkName]["WAVHeader"] = GetHeaderString(); // not sure if we want to nest JSON representation here rather?
```

Comment:
This was originally developed as a debug function. 

#34 has been logged and requires that the WAV header be reprented in a JSON format.

It should be implemented before this may be pulled.

Reply:
I don't think #34 is necessary, it's in the scope of this ticket and PR